combo_cases_and_deaths/delphi_combo_cases_and_deaths/run.py

```diff
@@ -112,30 +131,38 @@ def run_module():
         # create combined files for all of the historical reports
         params['date_range'] = [params['export_start_date'], yesterday]
     else:
-        pattern = re.compile(r'^\d{8}-\d{8}$')
-        match_res = re.findall(pattern, params['date_range'])
+        match_res = re.findall(re.compile(r'^\d{8}-\d{8}$'), params['date_range'])
         if len(match_res) == 0:
             raise ValueError(
                 "Invalid date_range parameter. Please choose from (new, all, yyyymmdd-yyyymmdd).")
         try:
             date1 = datetime.strptime(params['date_range'][:8], '%Y%m%d').date()
-        except ValueError:
-            raise ValueError("Invalid date_range parameter. Please check the first date.")
+        except ValueError as error:
+            raise ValueError(
+                "Invalid date_range parameter. Please check the first date.") from error
         try:
             date2 = datetime.strptime(params['date_range'][-8:], '%Y%m%d').date()
-        except ValueError:
-            raise ValueError("Invalid date_range parameter. Please check the second date.")
+        except ValueError as error:
+            raise ValueError(
+                "Invalid date_range parameter. Please check the second date.") from error
 
         #The the valid start date
         if date1 < params['export_start_date']:
             date1 = params['export_start_date']
         params['date_range'] = [date1, date2]
+    return params
 
-    for metric, geo_res, sensor_name, signal in variants:
 
-        df = combine_usafacts_and_jhu(signal, geo_res, extend_raw_date_range(params, sensor_name))
+def run_module():
+    """Produce a combined cases and deaths signal using data from JHU and USA Facts"""
+    variants = [tuple((metric, geo_res)+sensor_signal(metric, sensor, smoother))
+                for (metric, geo_res, sensor, smoother) in
+                product(METRICS, GEO_RESOLUTIONS, SENSORS, SMOOTH_TYPES)]
+    params = configure(variants)
+    for metric, geo_res, sensor_name, signal in variants:
+        df = combine_usafacts_and_jhu(signal, geo_res, extend_raw_date_range(params, sensor_name)) # pylint: disable=invalid-name
 
-        df = df.copy()
+        df = df.copy() # pylint: disable=invalid-name
```

Comment:
We should find a way to globally do this (not in the scope of this PR).

Reply:
can achieve this through a pylint rc

Reply:
@chinandrew yeah, though we typically only want to accept one or two otherwise invalid variable names and still check the rest

Reply:
Never used this functionality before, but pylint lets you [enter custom regex to filter invalid names](http://pylint.pycqa.org/en/latest/user_guide/options.html#custom-regular-expressions), so might be able to add certain ones as exceptions through that.